packages/core/helper-plugin/src/components/RelativeTime/index.js

```diff
@@ -18,18 +29,36 @@ const RelativeTime = ({ timestamp }) => {
 
   const relativeTime = isPast(timestamp) ? -interval[unit] : interval[unit];
 
+  // Display custom text if interval is less than the threshold
+  const customInterval = customIntervals.find((custom) => interval[custom.unit] < custom.threshold);
```

Comment:
```suggestion
  const customInterval = customIntervals.find((custom) => interval[custom.unit] <= custom.threshold);
```
Doesn't it sound more natural to have 1 to be less than an hour?

Reply:
I would say it feels more natural like this, we want the message only to appear if the interval is less than 60 minutes, if it's 70 minutes we want to display the actual interval difference. And if we were to do  `interval['hours'] =< 1` the message would be displayed for 2 hours.